actioncable/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "actioncable",
+  "version": "5.0.0-beta4",
+  "description": "WebSocket framework for Rails.",
```

Comment:
This is the description in `actioncable.gemspec` -- should I update that too? Wanted to try and keep things consistent to avoid confusion :)